c/src/ml-api-service.c

```diff
@@ -297,8 +300,22 @@ _ml_service_offloading_conf_to_opt (JsonObject * object, const gchar * name,
   for (iter = list; iter != NULL; iter = g_list_next (iter)) {
     const gchar *key = iter->data;
     const gchar *val = json_object_get_string_member (offloading_object, key);
-
     status = ml_option_set (option, key, g_strdup (val), g_free);
+
+    if (g_ascii_strcasecmp (iter->data, "training") == 0) {
+      /* Training offloading */
+      status =
+          ml_service_training_offloading_create (offloading_s,
+          offloading_object);
+      if (status != ML_ERROR_NONE) {
+        _ml_error_report_return (status,
+            "Failed to parse the configuration file.");
+      }
+      offloading_s->offloading_type = ML_OFFLOADING_TYPE_TRAINING;
+      continue;
+    }
+    val = json_object_get_string_member (offloading_object, iter->data);
+    status = ml_option_set (option, iter->data, g_strdup (val), g_free);
```

Comment:
Duplicated lines L302~L303. Pleas Remove these.

Reply:
It is changed it last pr. Thank you.